doc/articles/controls/TimePicker.md

```diff
@@ -56,7 +56,7 @@ If you wish to customize the overlay color, add the following to your top-level
 		Color="Pink" />
 ```
 
-Since **iOS14** the native `TimePicker` changed the way it's presented. By default iOS14 devices will display this new style.  You can still force the previous style (the one found in iOS13 or earlier) by adding the following at your `App.xaml.cs` class:
+Since **iOS14** the native `TimePicker` changed the way it's presented. By default iOS14 devices will display this new style.  You can still force the previous style (the one found in iOS13 or earlier) by adding the following at your `App.cs` or `App.xaml.cs` class:
```

Comment:
```suggestion
Since **iOS14** the native `TimePicker` changed the way it's presented. By default iOS14 devices will display this new style.  You can still force the previous style (the one found in iOS13 or earlier) by adding the following to your `App.cs` or `App.xaml.cs` class:
```